services/121-service/examples/fsp-anonymous1.json

```diff
@@ -0,0 +1,13 @@
+{
+  "fsp": "Bank A",
+  "attributes": [
+    {
+      "name": "idNumber",
+      "label": {
+        "en": "What is your national ID, so we can transfer money to your bank account"
+      },
+      "answerType": "text",
+      "option": null
```

Comment:
Does it need to be specified/included if it is `null`?

Reply:
Nope, but it does make it extra clear when we do want to add options